File: LE6/d8-1.py
a = float(input("Digite o lado 1: "))
b = float(input("Digite o lado 2: "))
c = float(input("Digite o lado 3: "))

if a == b and b == c:
    print("Triângulo Equilátero")
elif a == b or a == c or b == c:
    print("Triângulo Isósceles")
else:
    print("Triângulo Escaleno")

File: LE6/d8-1.java
import java.util.Scanner;

public class Ex8_V1 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        double a, b, c;

        System.out.print("Digite o lado 1: ");
        a = sc.nextDouble();

        System.out.print("Digite o lado 2: ");
        b = sc.nextDouble();

        System.out.print("Digite o lado 3: ");
        c = sc.nextDouble();

        if (a == b && b == c) {
            System.out.println("Triângulo Equilátero");
        } else if (a == b || a == c || b == c) {
            System.out.println("Triângulo Isósceles");
        } else {
            System.out.println("Triângulo Escaleno");
        }
    }
}